fastlane/lanes/screenshots.rb

```diff
@@ -64,8 +64,7 @@
   	rebuild_screenshot_devices
 
     # If you update this, be sure to also update the `rebuild_screenshot_devices` lane down below as well as the corresponding `fastlane/emulators/*.ini` config files.
-    # You might also want to mirror the changes of emulators used here with the ones used on Firebase Test Lab (when running on CI via `trigger_screenshots` lane),
-    # by updating the 2 places in the `raw-screenshots` job of `.circleci/config.yml` mentioning the devices used by FTL for screenshots
+    # You might also want to mirror the changes of emulators used here with the ones used on Firebase Test Lab (when running screenshots on CI)
```

Comment:
same as above: left this mention of "FTL running on CI" in the comment on purpose to not forget to keep a reminder when we'll rework this soon(ish)